ansible/roles/bastion/tasks/main.yml

```diff
@@ -38,7 +38,8 @@
 - name: Install mosh
   become: true
   yum:
-    name: https://gpte-public.s3.amazonaws.com/mosh-1.3.2-1.el7.x86_64.rpm
+    # name: https://gpte-public.s3.amazonaws.com/mosh-1.3.2-1.el7.x86_64.rpm
+    name: mosh
```

Comment:
is this always availble on the bastion with the default repos?
Or it via something like EPEL?

Reply:
Leaving this untouched in the next commit